build.fsx

```diff
@@ -31,6 +31,14 @@ Target "Clean" (fun _ ->
     CleanDirs [outDir]
 )
 
+let copyStylesheet() =
+    try
+        let cssFile = (slidesDir @@ "custom.css")
+        if (fileExists cssFile) then
```

Comment:
please add a default file instead. that will make it easier for people to understand what to do